plugins/@grouparoo/bigquery/__tests__/integration/app-refresh-query.ts

```diff
@@ -25,6 +25,10 @@ describe("integration/runs/bigquery/appRefreshQuery", () => {
     await app.update({ state: "ready" });
   });
 
+  afterEach(async () => {
+    await AppRefreshQuery.truncate();
+  });
```

Comment:
Not a problem, but the fixtures /probably/ didn't need to be re-recorded since all that changed in these tests was this (local only) cleanup method.